action_plugins/qubesformation.py

```diff
@@ -17,6 +17,13 @@ def generate_datastructure(vms, task_vars):
     dc = collections.OrderedDict
     d = dc()
     for n, data in vms.items():
+        # This block will skip any VMs that are not in the 'vms' variable
+        # This allows you to deploy in multiple stages which is useful in cases
+        # where you want to create a template after another template is already provisioned.
```

Comment:
That should not be necessary, the dependencies should already take care of deploying things in the right order.

Reply:
I see what you did here.  Merged.